vitrina/orgs/services.py

```diff
@@ -83,14 +86,15 @@ class Role(Enum):
     (Project, Action.CREATE): [Role.ALL],
     (Project, Action.UPDATE): [Role.AUTHOR],
     (Project, Action.DELETE): [Role.AUTHOR],
+    (Project, Action.VIEW): [Role.AUTHOR],
     (User, Action.UPDATE): [Role.AUTHOR],
```

Comment:
```suggestion
    (User, Action.UPDATE): (Role.AUTHOR),
```

Will save ton of bytes in ram. Optional

Reply:
4dbf96552d46a42e8f53fd1bdc860baa5697cdf5